.env.example

```diff
@@ -202,5 +215,13 @@ VITE_APP_TITLE="LibreChat"
 # and have set the GOOGLE_CLIENT_ID and GOOGLE_CLIENT_SECRET above
 VITE_SHOW_GOOGLE_LOGIN_OPTION=false
 
+# Enable OpenID Login
+# This enables/disables the Login with OpenID button on the login page.
+# Set to true if you have registered the app with an OpenID provider
+# and have set the OPENID_CLIENT_ID, OPENID_CLIENT_SECRET, OPENID_ISSUER, 
+# OPENID_SESSION_SECRET, and OPENID_SCOPE above
+ALLOW_OPENID=false
+OPENID_LABEL=OpenID
```

Comment:
Hey as mentioned in the previous PR, you need to have one of the vite enabled prefix's for this to pass through. ie `VITE_` `SCRIPT_` or one of the ones I have setup ie. `ALLOW_` (which is why the other one works).
I'm still not sure if a label is really needed anyway? will people really need to name it something other than OpenID?

Reply:
For our organization, we would not name it OpenID.  As a school, we give our credentials a unique name and reference that across various platforms. This way the user does not know the authentication system is LDAP, OpenID, Azure AD, or SAML. It provides consistency across various platforms.  I moved the pull to draft and will make the corrections on the name of the label and retest.